css/style.css

```diff
@@ -1 +1,38 @@
+body h1 h2 h3 h4	{
+	background-color: #f0efef;
+	font-family: 'Pavanam', serif;
+	padding: 20px;
+}
 
+h1 {
+font-size: 38px;
+font-weight: 700;
+}
+
+h2 {
+font-size: 24px;
+font-weight: 500;
+
+}
+
+h3 {
+font-size: 22px;
+font-weight: 400;
+
+}
+
+h4 {
+font-size: 16px;
+font-weight: 300;
+
+}
+
+ul	{
+	font-family: 'Pavanam', sans-serif;
```

Comment:
If you apply this at the top of your stylesheet to the body tag, then it will be applied to your entire page and you will not have to write it twice.